discovery-provider/src/tasks/index.py

```diff
@@ -127,6 +132,21 @@ def index_blocks(self, db, blocks_list):
             former_current_block.is_current = False
             session.add(block_model)
 
+            '''
+            # TEMPORARY TEST BY ADDING TWO Block entries
```

Comment:
@dmanjunath lmk if this test is sufficient, i was able to hit the below message with it

```
celery-worker_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
celery-worker_1  |     exc_info
celery-worker_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
celery-worker_1  |     reraise(type(exception), exception, tb=exc_tb, cause=cause)
celery-worker_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
celery-worker_1  |     raise value.with_traceback(tb)
celery-worker_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1170, in _execute_context
celery-worker_1  |     context)
celery-worker_1  |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 683, in do_executemany
celery-worker_1  |     cursor.executemany(statement, parameters)
celery-worker_1  | sqlalchemy.exc.IntegrityError: (psycopg2.IntegrityError) duplicate key value violates unique constraint "blocks_is_current_idx"
celery-worker_1  | DETAIL:  Key (is_current)=(t) already exists.
```

Reply:
@hareeshnagaraj this looks right. only one unique value true across the entire table